app/src/main/java/com/github/se/travelpouch/ui/home/Map.kt

```diff
@@ -52,6 +53,15 @@ fun MapContent(modifier: Modifier = Modifier, travelContainers: List<TravelConta
     }
   }
 
+  LaunchedEffect(travelContainers) {
+    if (travelContainers.isNotEmpty()) {
+      val firstLocation = travelContainers.first().location
+      cameraPositionState.position =
+          CameraPosition.fromLatLngZoom(
+              LatLng(firstLocation.latitude, firstLocation.longitude), 10f)
+    }
+  }
+
```

Comment:
Could you explain what this bit does? I'm a bit confused

Reply:
The map camera should display the location of the travels. however the travels take time to be fetched so the camera initializes to its default location. I added this code so that as soon as the list of travels changes (so they are fetch) the camera updates and positions itself on the first travels